src/d3d9/d3d9_common_texture.cpp

```diff
@@ -21,6 +21,13 @@ namespace dxvk {
       AddDirtyBox(nullptr, i);
     }
 
+    if (m_desc.Pool != D3DPOOL_DEFAULT) {
```

Comment:
We have observed initial upload always being true for buffers, is this also true for textures?

Reply:
I've seen games that rely on readonly uploads for textures (#1980 for example) so I think so.

Reply:
I've seen games that rely on readonly uploads for textures (#1980 for example) so I think so.